google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/Connection.java

```diff
@@ -632,6 +642,17 @@
    */
   Timestamp getCommitTimestamp();
 
+  /**
+   * @return the {@link CommitResponse} of the last {@link TransactionMode#READ_WRITE_TRANSACTION}
+   *     transaction. This method will throw a {@link SpannerException} if there is no last {@link
+   *     TransactionMode#READ_WRITE_TRANSACTION} transaction (i.e. the last transaction was a {@link
```

Comment:
per google style avoid Latin i.e. Perhaps rewrite as 

TransactionMode#READ_WRITE_TRANSACTION} transaction. That is, if the last transaction was a {@link

Reply:
Done.